data/json/hobbies.json

```
@@ -91,5 +94,368 @@
     "description": "You enjoyed camping in the vast wilderness as a means of escaping the pace of modern life.  The wilderness isn't filled with people, so it must be safe.  Right?  It must be.",
     "points": 2,
     "skills": [ { "level": 2, "name": "survival" }, { "level": 2, "name": "firstaid" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "hobby_heli_pilot",
+    "name": "Helicopter Pilot",
+    "description": "You got your pilot's license, and earned a living ferrying businessmen and tourists around.",
+    "points": 2,
+    "skills": [ { "level": 2, "name": "driving" }, { "level": 2, "name": "mechanics" } ],
+    "proficiencies": [ "prof_helicopter_pilot" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "loves_books",
+    "name": "Loves Books",
+    "description": "You love reading, you just never seemed to have enough time.  Maybe the cataclysm is a blessing in disguise.",
+    "points": 1,
+    "traits": [ "LOVES_BOOKS" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "chem_cooking",
+    "name": "Chemical Cooking",
+    "description": "You decided to cook your fix instead of waiting for your next one.",
+    "points": 2,
+    "proficiencies": [ "prof_intro_chemistry", "prof_intro_chem_synth" ],
+    "skills": [ { "level": 3, "name": "chemistry" }, { "level": 1, "name": "cooking" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "archery",
+    "name": "Archery",
+    "description": "You enjoy nocking back an arrow and hearing them whistle through the air.",
+    "points": 2,
+    "skills": [ { "level": 3, "name": "archery" }, { "level": 2, "name": "gun" } ],
+    "proficiencies": [ "prof_bow_basic" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "swimming",
+    "name": "Swimming",
+    "description": "Swimming through water is certainly an advantage you have over the undead.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "swimming" } ]
```

```suggestion
    "skills": [ { "level": 3, "name": "swimming" } ],
    "traits": [ "OUTDOORSMAN" ]
```
Another trait that can be hobbified(I'd also add it to Nomadic, but the wet protection applies more here)